### ISSUE TITLE
Console should not be used in test project which is using xunit 2.4(master branch)

### ISSUE THREAD

**Issue by chucklu** [kerryjiang/SuperSocket#188]
According this [documentation](https://xunit.github.io/docs/capturing-output)

> If you used xUnit.net 1.x, you may have previously been writing output to Console, Debug, or Trace. When xUnit.net v2 shipped with parallelization turned on by default, this output capture mechanism was no longer appropriate; it is impossible to know which of the many tests that could be running in parallel were responsible for writing to those shared resources. Users who are porting code from v1.x to v2.x should use one of the two new methods instead. 

You should not use Console to output string in xunit 2.4 .

**Comment by kerryjiang** [comment]
OK, I will adjust the code according this.

I had problems to use ITestOutputHelper before, so I switched to direct console output.

**Comment by kerryjiang** [comment]
Your pull request also will be welcome!

**Comment by chucklu** [comment]
Ok, I will create a new pull request later
